intg/src/main/java/org/apache/atlas/model/patches/AtlasPatch.java

```diff
@@ -49,6 +49,8 @@ public class AtlasPatch implements Serializable {
     private long        createdTime;
     private long        updatedTime;
     private PatchStatus status;
+    private long                assetsCountToPropagate;
+    private long                assetsCountPropagated;
```

Comment:
This was raised earlier https://github.com/atlanhq/atlas-metastore/pull/4032#discussion_r1942272635
This is not handles, why do we need this?